### PR TITLE
Day13 EX1 9Patch

### PR DESCRIPTION
**Pull request by at-hoavo**
#### Description (Required)
- [x] Create chatting screen use 9Patch

#### Reference (Optional)

### CODE REVIEW

**Comment by at-hoavo** [comment]
please check for me,thank you!

**Comment by at-tamnguyen2** [comment]
@at-hoavo em vẽ lại 9-path cho vùng hiển thị nội dung nó rộng ra tí, và set recycler view sao cho đến item dưới nếu ta submit send text thì recycler view nó tự trôi lên

File: app/src/main/java/vn/asiantech/internship/day13/ui/NinePathActivity.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        recyclerView.setAdapter(ninePathAdapter);
+        btnSend.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (!TextUtils.isEmpty(edtChat.getText())) {
+                    String text = edtChat.getText().toString();
+                    Random random = new Random();
+                    chats.add(new Chat(text, random.nextBoolean()));
+                    ninePathAdapter.notifyDataSetChanged();
+                    recyclerView.scrollToPosition(chats.size()-1);
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/day13/ui/NinePathActivity.java]: ERROR: '-' is not preceded with whitespace.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/day13/ui/NinePathActivity.java]: ERROR: '-' is not followed by whitespace.

**Comment by at-hoavo** [comment]
please check for me, thank you! @at-tamnguyen2

**Comment by at-tamnguyen2** [comment]
@at-hoavo 
LGTM
Thank you

**Comment by at-tienhoang** [comment]
@at-hoavo Please resolve conflict

**Comment by at-tienhoang** [comment]
@at-hoavo Please resolve conflict

**Comment by at-tienhoang** [comment]
LGTM! thank you
LGTM

**Pull request merged**